dns-resolver.cabal

```diff
@@ -47,6 +52,14 @@ library
   default-language:    Haskell2010
   ghc-options:         -Wall
 
+Executable cache-server
+  main-is:             server.hs
+  build-depends:       base
+                     , dns-resolver
+  hs-source-dirs:      mains
+  default-language:    Haskell2010
+  ghc-options:         -Wall -threaded -with-rtsopts=-N2 -with-rtsopts=-qn1
```

Comment:
This should be `-Wall -threaded -rtsopts` as users want to specify the number of cores by themselves.

Reply:
This is just the default option, so the user can still specify any options they wish.